src/Orckestra.Media.AutoImageResizing/ImageResizer.cs

```diff
@@ -60,12 +61,17 @@ private static void ConvertImageToPictureTag(XElement imgElement)
                     }
                 }
 
-                var altText = imgElement.Attributes().FirstOrDefault(item => item.Name.LocalName == "alt")?.ToString();
+                var altText = imgElement.Attributes().FirstOrDefault(item => item.Name.LocalName == "alt")?.Value;
+                var title = imgElement.Attributes().FirstOrDefault(item => item.Name.LocalName == "title")?.Value;
+                var id = imgElement.Attributes().FirstOrDefault(item => item.Name.LocalName == "id")?.Value;
 
                 pictureElement.Add(new XElement("img",
-                    new XAttribute("src", AutoImageResizingHelper.GetResizedImageUrl(imageSrc, MaxWidthLimit)),
+                    new XAttribute("src", AutoImageResizingHelper.GetResizedImageUrl(imageSrc, MaxWidth)),
                     string.IsNullOrWhiteSpace(altText) ? null : new XAttribute("alt", altText),
+                    string.IsNullOrWhiteSpace(title) ? null : new XAttribute("title", title),
+                    string.IsNullOrWhiteSpace(id) ? null : new XAttribute("id", id),
```

Comment:
It will be more future proof and safer to copy all of the attributes, instead of hardcoding "alt", "title" and "id". For example:

```
var attributesToCopy = imgElement.Attributes()
      .Where(_ => _.Name.LocalName != "src" 
             && _.Name.LocalName != "loading");

var imgElement = new XElement(ImgTagXName, .... );
imgElement.Add(attributesToCopy.Select(a => new XAttribute(a.Name, a.Value)));
```